core/blockly.js

```diff
@@ -65,6 +65,12 @@ goog.require('Blockly.Xml');
 var CLOSURE_DEFINES = {'goog.DEBUG': false};
 /* eslint-enable no-unused-vars */
 
+/**
+ * Blockly core version.
+ * @define {string}
+ */
+Blockly.VERSION = 'unset';
```

Comment:
I wonder if this should be 'uncompiled'...  Just a thought.  No opinion on this.